package-lock.json

```diff
@@ -8026,7 +8026,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```

Comment:
All navigation changes look good, but `package-lock.json` should not be updated?

Reply:
No idea how that happened, but fixed that now.
![isoladeifamosi-isola-isola14-isola2019-1Y64pa5uJW5nKMjYDm](https://media1.tenor.com/images/3d4d31e731895a886a9c5c241a607db2/tenor.gif?itemid=13597091)